lpa-submissions-e2e-tests/cypress/integration/appealsPlanningOfficersReport/appealsPlanningOfficersReportSteps.js

```diff
@@ -0,0 +1,125 @@
+import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps';
+
+const pageTitle =
+  'Upload report to reach decision - Appeal Questionnaire - Appeal a householder planning decision - GOV.UK';
+const pageUrl = '/officers-report';
+const sectionName = 'Required documents';
+const validFileName = 'upload-file-valid.pdf';
+const completedStatusId = 'officers-report';
+const questionTitle = 'Planning Officer’s report';
+
+Given('LPA Planning Officer is within the LPA Questionnaire task list', () => {
+  cy.goToAppealsQuestionnaireTasklistPage();
+});
+
+Given('Upload the Planning Officers report is requested', () => {
+  cy.goToPage(pageUrl);
+  cy.verifyPage(pageUrl);
+  cy.verifyPageTitle(pageTitle);
+  cy.verifyQuestionTitle(questionTitle);
+  cy.verifySectionName(sectionName);
+});
+
+Given('a file has been uploaded', () => {
+  cy.goToPage(pageUrl);
+  cy.clickChooseFile(validFileName);
+  cy.clickUploadFile().click();
+  cy.validateFileUpload(validFileName);
+});
+
+Given('a file has been uploaded and confirmed', () => {
+  cy.goToUploadPlanningOfficersReportPage();
+  cy.clickChooseFile(validFileName);
+  cy.clickUploadFile().click();
+  cy.validateFileUpload(validFileName);
+  cy.clickSaveAndContinue();
+});
+
+Given('The question "Upload the Planning Officers report" has been completed', () => {
+  cy.goToPage(pageUrl);
+  cy.clickChooseFile(validFileName);
+  cy.clickUploadFile().click();
+  cy.validateFileUpload(validFileName);
+  cy.clickSaveAndContinue();
+});
+
+When('LPA Planning Officer deletes the file', () => {
+  cy.deleteUploadedFile(validFileName);
```

Comment:
Do we need an API request to check if the file exists before we delete it?

Reply:
No, this only test the UI process, not actual deletion.